### DISCUSSION TITLE
Share static props between pages

### DISCUSSION THREAD

**Discussion by franciscorubin** [vercel/next.js#16408] [Help]
I would like to have the ability to share static props obtained through getStaticProps between multiple pages.

In my case, I have a dynamic route with around 20 different pages that fetches exactly the same data from a Salesforce API. I would like to be able to fetch it just once to reduce my build times and API quotas spent.

**Comment by meandillar** [comment]
I have the same issue. There are around 2000 products on our store which need their own page generated like `/products/[product]`, but each product is also listed in multiple collections like `/collections/[collection]/products/[product]` resulting in around 25k pages generated many of which are fetching the same data in getStaticPaths and getStaticProps

**Comment by franciscorubin** [comment]
Why has this been moved to Discussions?? It's not a discussion, it's a feature request. Does this indicate that you have no plans of implementing this feature? I would appreciate at least a comment saying that, instead of keeping us in the dark. Creating issues requires effort, and there is zero motivation of expending it if they are hidden randomly without an explanation (this is the second time this happens to me in this repository, the first one was a bug report that I still haven't solved: https://github.com/vercel/next.js/discussions/15932).

@arunoda 
@timneutkens 
@nkzawa 
@rase- 
@rauchg

**Reply by timneutkens** [reply]
If the issue template is completely ignored (which happened to you in both cases) it will be automatically converted to a discussion. https://github.com/vercel/next.js/issues/new/choose

There is already feature request open here: https://github.com/vercel/next.js/issues/12434

**Reply by franciscorubin** [reply]
Good to know, I thought the templates were a suggestion rather than a requirement.

Thanks for pointing in the right direction.
